### PR TITLE
[#IOCIT-162] Added type for ulid strings

### PR DESCRIPTION
**Pull request by michaeldisaro**
#### Types of changes

- [x] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/__tests__/strings.test.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import * as t from "io-ts";
 
 import { isLeft, isRight } from "fp-ts/lib/Either";
-import { PatternString, Semver } from "../strings";
+import { PatternString, Semver, UlidString } from "../strings";
```
[the remark this answers is not in the document]
Comment (michaeldisaro) [reply src/__tests__/strings.test.ts:4]: You're right, I called it Ulid at the beginning, but I was a bit worried about the fact that generally we import `ulid` the only difference would be the case. I'd like some other opinion about this.

[the remark this answers is not in the document]
Comment (michaeldisaro) [reply src/__tests__/strings.test.ts]: Ok, did it in https://github.com/pagopa/io-ts-commons/pull/404/commits/15874db485c4eaeef61b40a44d71c5b39f490b54

**Review by arcogabbo — APPROVED** [review]
LGTM

**Review by BurnedMarshal — APPROVED** [review]

File: src/strings.ts [hunk 2]
```diff
@@ -163,3 +163,10 @@ export const Semver = PatternString(
 );
 
 export type Semver = t.TypeOf<typeof Semver>;
+
+/**
+ * A valid ULID string
+ */
+export const Ulid = PatternString("^[0-9A-HJKMNP-TV-Z]{26}$");
```
Comment (fabriziopapi) [inline src/strings.ts:170]: ULID should be case insensitive

Comment (michaeldisaro) [reply src/strings.ts]: fixed in https://github.com/pagopa/io-ts-commons/pull/404/commits/7171960c8ea51acad761cbb9d51010ae4848166f

**Pull request merged**